### PR TITLE
[capitaldao-pool] Capital Dao Staking Pool

### PR DESCRIPTION
**Pull request by diepvuongthang**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/strategies/capitaldao-pool/index.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        [options.poolIndex]
+      ])
+    ],
+    { blockTag }
+  );
+  
+  return Object.fromEntries(
+    response.map((pool, i) => {
+      const parsedAmount = parseFloat(
+        formatUnits(bn(pool.totalStaked), options.decimal)
```
Comment (ChaituVR) [inline src/strategies/capitaldao-pool/index.ts:44]: Hi @diepvuongthang Did you try using https://snapshot.org/#/strategy/contract-call ? you can pass `totalStaked` into `output` param

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/capitaldao-pool/index.ts (hunk 1 above)
Comment (diepvuongthang) [reply src/strategies/capitaldao-pool/index.ts:44]: const strategies = [
  ["contract-call", {
    // contract address
    "address": "0x2613D723F64E45Bade6aD8BceBAa31496b9f8b74",
    // output decimals
    "decimals": 18,
    // strategy symbol
    "symbol": "CDS",
    // arguments are passed to the method; "%{address}" is replaced with the voter's address; default value ["%{address}"]
    "args": [0], 
    // method ABI, output type should be uint256
    "methodABI": {
      "inputs":[{"internalType":"uint256","name":"","type":"uint256"}],
      "name":"pools",
      "outputs":[{"internalType":"contract IERC20","name":"token","type":"address"},
      {"internalType":"uint256","name":"allocPoint","type":"uint256"},
      {"internalType":"uint256","name":"lastRewardBlock","type":"uint256"},
      {"internalType":"uint256","name":"rewardPerShare","type":"uint256"},
      {"internalType":"uint256","name":"lockTime","type":"uint256"},
      {"internalType":"uint256","name":"totalStaked","type":"uint256"}],
      "stateMutability":"view","type":"function"
    }
  }],
]
I try it, it don't work. can you check it @ChaituVR ?

Comment (ChaituVR) [reply src/strategies/capitaldao-pool/index.ts:44]: Hi @diepvuongthang I mean like this 
https://snapshot.org/#/playground/contract-call?query=eyJwYXJhbXMiOnsiYWRkcmVzcyI6IjB4MjYxM0Q3MjNGNjRFNDVCYWRlNmFEOEJjZUJBYTMxNDk2YjlmOGI3NCIsInN5bWJvbCI6IkNEUyIsImRlY2ltYWxzIjoxOCwiYXJncyI6WyIwIl0sIm91dHB1dCI6InRvdGFsU3Rha2VkIiwibWV0aG9kQUJJIjp7ImlucHV0cyI6W3siaW50ZXJuYWxUeXBlIjoidWludDI1NiIsIm5hbWUiOiIiLCJ0eXBlIjoidWludDI1NiJ9XSwibmFtZSI6InBvb2xzIiwib3V0cHV0cyI6W3siaW50ZXJuYWxUeXBlIjoiYWRkcmVzcyIsIm5hbWUiOiJ0b2tlbiIsInR5cGUiOiJhZGRyZXNzIn0seyJpbnRlcm5hbFR5cGUiOiJ1aW50MjU2IiwibmFtZSI6ImFsbG9jUG9pbnQiLCJ0eXBlIjoidWludDI1NiJ9LHsiaW50ZXJuYWxUeXBlIjoidWludDI1NiIsIm5hbWUiOiJsYXN0UmV3YXJkQmxvY2siLCJ0eXBlIjoidWludDI1NiJ9LHsiaW50ZXJuYWxUeXBlIjoidWludDI1NiIsIm5hbWUiOiJyZXdhcmRQZXJTaGFyZSIsInR5cGUiOiJ1aW50MjU2In0seyJpbnRlcm5hbFR5cGUiOiJ1aW50MjU2IiwibmFtZSI6ImxvY2tUaW1lIiwidHlwZSI6InVpbnQyNTYifSx7ImludGVybmFsVHlwZSI6InVpbnQyNTYiLCJuYW1lIjoidG90YWxTdGFrZWQiLCJ0eXBlIjoidWludDI1NiJ9XSwic3RhdGVNdXRhYmlsaXR5IjoidmlldyIsInR5cGUiOiJmdW5jdGlvbiJ9fSwibmV0d29yayI6IjQiLCJzbmFwc2hvdCI6MTAzODM4MDQsImFkZHJlc3NlcyI6WyIweDAwOWNGN2JDNTc1ODRiNzk5ODIzNmVmZjUxYjk4QTE2OERjZUE5QjAiXX0.

Comment (diepvuongthang) [reply src/strategies/capitaldao-pool/index.ts:44]: thank you @ChaituVR

**Pull request closed without merging**